benches/noria/run.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/bash
+
```

Comment:
in general, unless there's a reason not to, i think most bash scripts should probably begin with 
```bash
set -o nounset
set -o errexit
```
(or `set -eu`)

Reply:
It means that
```bash
cmd
if [[ $? -ne 0 ]]; then
fi
```
type code won't work, but I worked around it :)